org.scala-ide.sdt.debug/src/org/scalaide/debug/internal/model/JdiRequestFactory.scala

```diff
@@ -78,8 +79,7 @@ object JdiRequestFactory {
     threadStartRequest
   }
 
-  /**
-   * Create thread death request
+  /** Create thread death request
    */
   def createThreadDeathRequest(virtualMachine: VirtualMachine): ThreadDeathRequest = jdiSynchronized {
     val threadStartRequest = virtualMachine.eventRequestManager.createThreadDeathRequest
```

Comment:
not for now, but all of these methods need eventRequestManager so maybe it should be exposed somehow?